frontend/app/model/Benefits.scala

```diff
@@ -1,75 +1,56 @@
 package model
 
 import com.gu.membership.salesforce.Tier
-import com.gu.membership.salesforce.Tier.{Partner, Patron, Staff}
+import com.gu.membership.salesforce.Tier._
 import configuration.Config.zuoraFreeEventTicketsAllowance
 
-object Benefits {
-
-  val DiscountTicketTiers = Set[Tier](Staff, Partner, Patron)
-  val ComplimenataryTicketTiers = Set[Tier](Partner, Patron)
+case class Benefits(tier: Tier, pricing: Option[Pricing]) {
```

Comment:
We should probably separate pricing away from Benefits, because are benefits shouldn't vary whether the user is a test-user or not.